visualizations/bar_chart/webviz_bar_chart/__init__.py

```diff
@@ -13,19 +13,22 @@ class BarChart(FilteredPlotly):
         ``'overlay'``. Defines how multiple bars per index-value are combined.
         See `plotly.js layout-barmode <https://plot.ly/javascript/reference/
         #layout-barmode>`_.
-    :param kwargs: optional `xaxis` and `yaxis` paramameter. Will create a
-        label for the given axis. Defaults to `None`.
+    :param xaxis: Will create a label for the x-axis. Defaults to `None`.
+    :param yaxis: Will create a label for the y-axis. Defaults to `None`.
+    :param logy: boolean value to toggle y-axis logarithmic scale.
+        Defaults to False
```

Comment:
→ Change from False to `False` (i.e. with back ticks) to get same style as with `None` above?